HACKING.md

```diff
@@ -1,69 +1,77 @@
+Building on Windows
+---------------
+You will require the following installed on your computer: Git and Visual Studio 2019/2022.
 
-* Clone, fork or download the repo "https://github.com/gp-alex/world-of-might-and-magic.git"
+* Clone, fork or download the repo `https://github.com/gp-alex/world-of-might-and-magic.git`
 * Open the folder in Visual Studio
-* Select build configuration (x32 or x64) and wait for CMake cache to complete
-* Select startup item as World of Might and Magic
+* Select build configuration (x32 or x64) and wait for CMake configuration to complete
+* Select startup item as `World_of_Might_and_Magic.exe`
 * Run!
 
-Visual studio has a bug with git submodules not syncing between branches.
-When checking out the branch you may need to run the following command: "git submodule update --init"
-
-MSVC users please note this project requires Windows SDK 10.0.20348.0 or higher
+__Be aware__ that Visual Studio has a bug with git submodules not syncing between branches.
+So when checking out the branch or switching to different branch you may need to run the following command manually: `git submodule update --init`
 
-Still having problems? Try the discord chat [![](https://img.shields.io/badge/chat-on%20discord-green.svg)](https://discord.gg/jRCyPtq)
+Support
+---------------
+Still having problems? Try to ask for help on our discord. [![](https://img.shields.io/badge/chat-on%20discord-green.svg)](https://discord.gg/jRCyPtq)
 
 
 Coding style
-------------
-For the C++ code we are following the [Google C++ Style Guide](http://google.github.io/styleguide/cppguide.html).<br />
-Source code is automatically checked against it, and Pull Request will fail if you don't follow it.<br />
-For style check on Windows platform, you can use [Visual Studio Code cpplint plugin](https://marketplace.visualstudio.com/items?itemName=mine.cpplint).
+---------------
+For the C++ code we are following the [Google C++ Style Guide](http://google.github.io/styleguide/cppguide.html).\
+Source code is automatically checked against it and Pull Request will fail if you don't follow it.
 
-In visual studio, style can be checked (Python requred). Go to Solution Explorer->Change Views->CMake targets
-Right click and build check_style, errors will be listed in output
+To perform style check before pushing anything you can build `check_style` target. In Visual Studio you can do that by going to ***Solution Explorer → Change Views → CMake targets***. Right click and build `check_style`, errors will be listed in output.
 
 Some additional style preferences that we follow, in no particular order:
 * `*` and `&` in type declarations are preceded by a space. So it's `char *string`, and not `char* string`.
-* Documentation should be in doxydoc format with `\` used for tags, and starting with `/**` comment introducer.
+* Documentation should be in doxydoc format with `@` used for tags, and starting with `/**` comment introducer.
```

Comment:
Why @ though?

Reply:
Currently we are using @ in special commands everywhere… Or tags mentioned there are something else?